scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -113,21 +48,18 @@ class ThemeAggregator {
    * @returns {array} - An array of aggregated theme files
    */
   static aggregateThemes(options) {
-    if (!ThemeAggregator.validate(options)) {
-      return null;
-    }
+    if (!ThemeAggregator.validate(options)) return null;
 
-    // Create generated-themes directory.
     fs.ensureDir(OUTPUT_DIR, (err) => {
-      Logger.warn(err, { context });
+      Logger.warn(err, { LOG_CONTEXT });
```

Comment:
If this doesn't exist, do we create it somewhere or return null?

Reply:
ensureDir creates the directory if it does not exist - if it exists, I don't think it does anything